index.html

```diff
@@ -9,47 +9,154 @@
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+  
+        <div class="proj-1">
+          <img src="static/Proj5.png" alt="project 5" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+  
+        <div class="proj-1">
+          <img src="static/Proj6.png" alt="project 6" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+      </section>
+  
+      <section class="about">
+        <div class="about-description">
+          <h2>About</h2>
+          <p>
+            I can help you build a product , feature or <br />
+            website Look through some of my work <br />
+            and experience! If you like what you see <br />
+            and have a project you need coded, <br />
+            don’t hesitate to contact me.
+          </p>
+          <button type="button">
+            Get my CV <img src="static/ic_download.svg" />
+          </button>
+        </div>
+        <div class="languages-container">
+          <ul class="languages">
+            <li>Languages</li>
+            <li>JavaScript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+          <div class="hashtag"></div>
+        </div>
+  
+        <div class="frameworks-container">
+          <ul class="frameworks">
+            <li>Frameworks</li>
+            <li>React.js</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+          </ul>
+          <div class="rectangle"></div>
+        </div>
+  
+        <div class="skills-container">
+          <ul class="skills">
+            <li>Skills</li>
+            <li>Database Management</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Version Control</li>
+            <li>Web Development</li>
+          </ul>
+          <div class="idea"></div>
+        </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li`. Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`. It is basically a nested `ul` with its own `li` in each of the `li` of the outer `ul`. Kindly fix this as this improve the accessibility of the page mostly for blind users who depends on using the keyboard `tab` key to navigate the webpage.